### PR TITLE
[Done] Ability to drag item into LONG LIST

### PR DESCRIPTION
**Pull request by bhavinsen**

### CODE REVIEW

**Review by chrisns — CHANGES_REQUESTED** [review]
indentation, and other linting errors
run `eslint .` and `npm run build` and fix issues, then I'll review the actual change

File: src/pages/Home.vue [hunk 1]
```diff
@@ -107,6 +120,25 @@ watch(() => props.podcastId, connect, {
   immediate: true,
 })
 
+	const draggedLongList = (x: number, y: number, item: Item, podcastId: string) => {
```
Comment (chrisns) [inline src/pages/Home.vue:123]: indentation?

**Review by chrisns — CHANGES_REQUESTED** [review]
appears to be built on top of #154 so either that'll need to be merged before this can be, or this will need to be refactored to be independent

File: src/pages/Home.vue (hunk 1 above)
Comment (bhavinsen) [reply src/pages/Home.vue:149]: No

File: src/pages/Home.vue [hunk 2]
```diff
@@ -133,6 +136,25 @@ watch(() => props.podcastId, connect, {
...
```
Comment (chrisns) [inline src/pages/Home.vue:139]: Indentation still off

File: src/pages/Home.vue [hunk 3]
```diff
@@ -133,6 +136,25 @@ watch(() => props.podcastId, connect, {
...
+  const id = 
+  <string>// @ts-ignore
+  document.elementFromPoint(x, y)?.attributes["item-id"]?.value
+  const slot = <Item["slot"]>parseInt(<string>// @ts-ignore
+  document.elementFromPoint(x, y)?.attributes["item-slot"]?.value)
```
Comment (chrisns) [inline src/pages/Home.vue:144]: indentation here too is off

File: src/pages/Home.vue [hunk 4]
```diff
@@ -133,6 +136,25 @@ watch(() => props.podcastId, connect, {
...
+  if(slot && slot == item.slot){
+    const slotItem =itemStore.getList
```
Comment (chrisns) [inline src/pages/Home.vue:146]:
```suggestion
    const slotItem = itemStore.getList
```

**Review by chrisns — CHANGES_REQUESTED** [review]

**Comment by chrisns** [comment]
Looks like there's basic code consistency errors that eslint should probably fix for you

**Pull request merged**